pkgs/top-level/all-packages.nix

```diff
@@ -26141,6 +26141,8 @@ with pkgs;
 
   git-review = python3Packages.callPackage ../applications/version-management/git-review { };
 
+  git-team = callPackage ../applications/version-management/git-and-tools/git-team {};
```

Comment:
```suggestion
  git-team = callPackage ../applications/version-management/git-and-tools/git-team { };
```

Reply:
Hmm, okay, there are 99 occurrences of {} in this file already. Is there a style guide available that covers this? I mean, I don't know why to choose one over the other yet.

Reply:
> Hmm, okay, there are 99 occurrences of {} in this file already

And a *lot* more of `{ }`

> Is there a style guide available that covers this?

- Don't think so, seems it just gets passed down with reviews like this one. 
- You can use `nixpkgs-fmt` for newer files.

Reply:
Corrected it. Actually I'm already using nixpkgs-fmt. It's probably a good idea to run it over the whole file and put that in a .git-blame-ignore-revs file

Reply:
I'd agree if we didn't have at least 878 [open PRs for new packages](https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+package+%28new%29%22). Merge conflicts would not be fun.